docs/faq.rst

```diff
@@ -319,7 +319,20 @@ I'm scraping a XML document and my XPath selector doesn't return any items
 
 You may need to remove namespaces. See :ref:`removing-namespaces`.
 
+How to fix ``AttributeError`` after creating a new spider or module?
+--------------------------------------------------------------------
+
+If you have recently created a new spider (e.g. with ``scrapy genspider``) or
+added a new module or package to your Scrapy project, and you get an
+``AttributeError`` exception at run time, make sure that the filename of your
```

Comment:
in https://github.com/scrapy/scrapy/issues/2680 output example doesn't have AttributeError. I also tried it locally, and got this:

```
> scrapy genspider os foo.com
Created spider 'os' using template 'basic' in module:
  splash_render.spiders.os
> scrapy list
bbox
os
random_follow
> scrapy runspider os
2019-03-14 22:13:51 [scrapy.utils.log] INFO: Scrapy 1.6.0 started (bot: splash_render)
2019-03-14 22:13:51 [scrapy.utils.log] INFO: Versions: lxml 4.2.5.0, libxml2 2.9.8, cssselect 1.0.3, parsel 1.5.1, w3lib 1.19.0, Twisted 17.9.0, Python 3.6.1 (default, Apr  4 2017, 09:40:21) - [GCC 4.2.1 Compatible Apple LLVM 8.1.0 (clang-802.0.38)], pyOpenSSL 17.5.0 (OpenSSL 1.1.0g  2 Nov 2017), cryptography 2.1.4, Platform Darwin-18.2.0-x86_64-i386-64bit
Usage
=====
  scrapy runspider [options] <spider_file>

runspider: error: File not found: os
```

Does an error depend on a command?

Reply:
Looks like it will give a different error depending on the module:

```
$ scrapy genspider csv bar.com
Created spider 'csv' using template 'basic'
$ scrapy runspider csv
Traceback (most recent call last):
  File "/home/adrian/.local/share/virtualenvs/test-Gyt2xk5R/bin/scrapy", line 6, in <module>
    from scrapy.cmdline import execute
  File "/home/adrian/.local/share/virtualenvs/test-Gyt2xk5R/lib/python3.6/site-packages/scrapy/__init__.py", line 34, in <module>
    from scrapy.spiders import Spider
  File "/home/adrian/.local/share/virtualenvs/test-Gyt2xk5R/lib/python3.6/site-packages/scrapy/spiders/__init__.py", line 130, in <module>
    from scrapy.spiders.feed import XMLFeedSpider, CSVFeedSpider
  File "/home/adrian/.local/share/virtualenvs/test-Gyt2xk5R/lib/python3.6/site-packages/scrapy/spiders/feed.py", line 8, in <module>
    from scrapy.utils.iterators import xmliter, csviter
  File "/home/adrian/.local/share/virtualenvs/test-Gyt2xk5R/lib/python3.6/site-packages/scrapy/utils/iterators.py", line 2, in <module>
    import csv
  File "/home/adrian/temporal/test/csv.py", line 5, in <module>
    class CsvSpider(scrapy.Spider):
AttributeError: module 'scrapy' has no attribute 'Spider'
```

Reply:
Oh, I was not using a project, that must explain the differences.

Reply:
Hey @Gallaecio! I haven't checked it recently, but the error message in the docs don't match message which I was getting - do you know the reason?